### PR TITLE
@v1.16.1 - Update github actions config

### PR DESCRIPTION
**Pull request by jamieomaguire**
v1.16.1
------------------------------
*October 13, 2022*

### Changed
- add more granular config to the GitHub actions file for running on pushes to main and PRs

### CODE REVIEW

**Review by siggerzz — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+      - '.husky/**'
+      - 'stories/**'
+      - '.vscode/**'
+      - 'README.md'
+      - 'CONTRIBUTING.md'
+      - 'CHANGELOG.md'
+      - 'LICENSE'
+  push:
+    branches:
+      - master
```
Comment (ashleynolan) [inline .github/workflows/build.yml:16]: Should this be `main`?

Comment (jamieomaguire) [reply .github/workflows/build.yml:16]: yes 🤣

Comment (jamieomaguire) [reply .github/workflows/build.yml]: good spot! updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v1.16.1
+------------------------------
+*October 13, 2022*
+
+### Changed
+- add more granular config to the GitHub actions file for running on pushes to master and PRs
```
Comment (ashleynolan) [inline CHANGELOG.md:12]:
```suggestion
- add more granular config to the GitHub actions file for running on pushes to main and PRs
```

**Pull request merged**